EpisodeUtilityView.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  EpisodeCellControlView.swift
+//  Recast
+//
+//  Created by Jack Thompson on 10/21/18.
+//  Copyright © 2018 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class EpisodeUtilityView: UIView {
+
+    // MARK: - Variables
+    var playButton: UIButton!
+    var downloadButton: UIButton!
+
+    // TODO: later change to download status to indicate episode is downloading
+    var isDownloaded: Bool = false
+
+    init(frame: CGRect, isDownloaded: Bool) {
+        super.init(frame: frame)
+
+        playButton = UIButton()
+        playButton.setTitle("Play", for: .normal)
+        playButton.titleLabel?.font = .systemFont(ofSize: 14)
+        playButton.setTitleColor(.gray, for: .normal)
+        playButton.setImage(#imageLiteral(resourceName: "play_icon"), for: .normal)
+
+        downloadButton = UIButton()
+        downloadButton.titleLabel?.font = .systemFont(ofSize: 14)
+        downloadButton.setTitleColor(.gray, for: .normal)
+
+        if isDownloaded {
+            downloadButton.setTitle("Downloaded", for: .normal)
```

Comment:
```suggestion
            downloadButton.setTitle(isDownloaded ? "Downloaded" : "Download", for: .normal)
```
You can combine this if/else into two lines